### PR TITLE
Set powershell-indent based off tab-width.

### PR DESCRIPTION
**Pull request by ralesi**
This will allow tab spacing to be consistent with the users existing setup.

### CODE REVIEW

**Comment by StreakyCobra** [comment]
Maybe this list can be used to set this? https://github.com/syl20bnr/spacemacs/blob/develop/layers/+distribution/spacemacs-bootstrap/config.el#L13

File: layers/+lang/windows-scripts/packages.el [hunk 1]
```diff
@@ -53,6 +53,8 @@
         (spacemacs|define-text-object "$" "dollarparen" "$(" ")")
         )
       (add-hook 'powershell-mode-hook 'powershell/define-text-objects)
+      ;; set powershell indent to tab-width
+      (setq powershell-indent tab-width)
```
Comment (bmag) [inline layers/+lang/windows-scripts/packages.el]: Doesn't it matter that `tab-width` is buffer-local? There's `(default-value 'tab-width)` if using the default value (set by `setq-default`) makes more sense here. (I don't actually know what's appropriate here, just raising a concern)

**Comment by ralesi** [comment]
@StreakyCobra,  this alist will probably work fine,  I will update this to give it a try.

**Comment by ralesi** [comment]
@StreakyCobra, this alist is used only for evil-shift-width, but I have updated including this change as well.

**Comment by StreakyCobra** [comment]
@ralesi Oh right, but indeed it's nice to had it there too. (And thanks for the key-bindings documentation update on your other PRs, I didn't want to spam you, so doing it here ;-) )

**Pull request closed without merging**